Suggested changes to clarify scope and deliverables, hoping to address M Boucadair's comments (#58)
OR13/diem-draft-charter: ocm/charter.md

OpenCloud, ownCloud, and Seafile. This working group seeks to formally

> specify OCM.

Reviewer: I would remove all product names.

Reviewer 2: No particular objections here. They were added in our first write up just for historical reasons.

A core use case of OCM is when a user with file-sharing functionality
within one administrative domain wishes to provide file access to a 
user in another administrative domain without transferring the resource itself.

> When many administrative domains are involved (e.g. many universities

Reviewer: ```suggestion
When many administrative domains are involved (e.g., many universities
```

and research centers with cross-collaboration) it becomes hard to allow
users to log into each others' administrative domains. Instead, a 

> federated system has been built that allows servers in one domain to

Reviewer: ```suggestion
federated system has been built to allow servers in one domain to
```

connect to trusted servers in other domains and make shared data 
available to their own logged-in users.

OCM defines the exchange up to the point where other established
protocols such as WebDAV or JMAP can take over. This layered design

> makes OCM flexible and agnostic to the underlying data exchange

Reviewer: ```suggestion
makes OCM agnostic to the underlying data exchange
```

mechanisms, enabling seamless integration between diverse systems.

The OCM specification has now grown
to a level of maturity where it will be useful to converge on a set of
features in a stable specification for vendors to build to or update to.

The architectural context for OCM is therefore federated servers already
providing an interoperable interface within their individual contexts, 
but requiring share mechanisms that cross administative boundaries, using
pre-existing trust mechanisms.

# Scope

> The OCM WG will produce a standards-track specification or family of

Reviewer: ```suggestion
The OCM WG will produce Standards Track specification(s)
```

> specifications, for OCM's share/invite functionality. With these 

Reviewer: ```suggestion
for OCM's share/invite functionality. With these 
```

OCM does not aim to address identity federation (e.g., OIDC or SAML);
rather, it leverages existing protocols for identity and authentication
where appropriate.  OCM also does not propose to create interoperable 
protocol flows to establish trust relationships where there are none 
before; instead the WG will explain how trust relationships set up 

> out-of-band (i.e. configuration of server settings or allow lists) 

Reviewer: ```suggestion
out-of-band (e.g., configuration of server settings or allow lists) 
```

are required for secure sharing.

> Importantly, OCM is **not** a competitor to user-oriented messaging or

Reviewer: Do we really need to have this text here?

Reviewer 3: This is done to note what is out-of-scope. I do think we can reword to be "OCM is not a user-oriented..."

Reviewer: Rewording as you suggested @anewton1998 works for me. I was initially puzzled why specific "social ..." protocols were listed.

# Deliverables

* One specification or family of specifications, that describe
OCM Invite/Share functionality and associated considerations such as

> security considerations.

Reviewer: ```suggestion
operational and security considerations.
```
